### DISCUSSION TITLE
Support for Lima VM

### DISCUSSION THREAD

**Discussion by templeman** [spatie/wordpress-ray#68] [Q&A] [answered]
I've been using Ray successfully with WordPress on Vagrant with VirtualBox. I recently switched over to [Lima](https://lima-vm.io/) to drive my local dev instance, and Ray calls are no longer registering in the app. Wondering if there are changes I need to make to my config to get Ray working with Lima.

My `ray.php` config

```
<?php
return [
    /*
     * Whether data should be sent to Ray.
     */
    'enable' => true,

    /*
     * The host used to communicate with the Ray app.
     */
    'host' => '10.0.2.2',

    /*
     * The port number used to communicate with the Ray app.
     */
    'port' => 23517,

    /*
     * Absolute base path for your sites or projects in Homestead, Vagrant, Docker, or another remote development server.
     */
    'remote_path' => null,

    /*
     * Absolute base path for your sites or projects on your local computer where your IDE or code editor is running on.
     */
    'local_path' => null,

    /*
     * When this setting is enabled, the package will not try to format values sent to Ray.
     */
    'always_send_raw_values' => false,
];
```

I tried changing `host` to `localhost` and adjusting the `port` value with no luck. I should mention I'm using Lima in conjunction with the Roots [Trellis](https://roots.io/trellis/) framework.

**Ray WP Plugin 1.7.5
Ray App 2.3.0
WP 6.1.1
PHP 8.0**

**Comment by freekmurze** [comment]
Hi,

thanks for using Ray.

Unfortunately, I don't have any experience with Lima.

The Ray package tries to send messages to the Ray desktop app via post 23517. Make sure that you're forwarding the port correctly from your virtual machine.

**Comment by templeman** [comment, marked as answer]
I got this working with some guidance from the [WordPress Roots community](https://discourse.roots.io/t/port-forwarding-with-lima-vm-and-ray-app/25167).

Setting `'host' => 'host.lima.internal'` is the only change needed for Ray to communicate with Lima VM. 🎉
